app/src/lib/stores/app-store.ts

```diff
@@ -109,6 +109,15 @@ import { Owner } from '../../models/owner'
 import { PullRequest } from '../../models/pull-request'
 import { PullRequestUpdater } from './helpers/pull-request-updater'
 
+/**
+ * Enum used by fetch to determine if
+ * a fetch was initiated by the backgroundFetcher
+ */
+export enum FetchType {
+  BackgroundTask,
+  NonBackgroundTask,
```

Comment:
:lipstick: naming things is Hard™, but could we make this name a bit more distinct? Something like `UserInitiated` will help it to stand out in code when we need to dig deeper into these flows.

For example, instead of this:

```ts
if (fetchType !== FetchType.BackgroundTask) {
  this._refreshPullRequests(repository)
}
```

It could become this:

```ts
if (fetchType === FetchType.UserInitiated) {
  this._refreshPullRequests(repository)
}
```